packages/common/source/utils/public-client.ts

```diff
@@ -32,6 +32,18 @@ const LOGIN_WITH_GOOGLE = gql`
     }
 `;
 
+const GET_GOOGLE_AUTH_SERVICES = gql`
+    query GetAppByName($name: String) {
+        getAppByName(name: $name) {
+            authServices {
+                googleAuth {
+                    clientId
+                }
+            }
+        }
+    }
+`;
+
```

Comment:
If `AppModel` has all the auth info, we might as well just create a function called `getAppByName` and extract the auth info from the return value.

Reply:
We should make it public.